core/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/TermsAggregatorFactory.java

```diff
@@ -69,8 +69,8 @@ public TermsAggregatorFactory(String name, Type type, ValuesSourceConfig<ValuesS
     @Override
     protected Aggregator createUnmapped(Aggregator parent, List<PipelineAggregator> pipelineAggregators, Map<String, Object> metaData)
             throws IOException {
-        final InternalAggregation aggregation = new UnmappedTerms(name, order, config.format(), bucketCountThresholds.getRequiredSize(),
-                bucketCountThresholds.getShardSize(), bucketCountThresholds.getMinDocCount(), pipelineAggregators, metaData);
+        final InternalAggregation aggregation = new UnmappedTerms(name, order, bucketCountThresholds.getRequiredSize(),
```

Comment:
I am a little wary of removing too much stuff here, for example the format string. I don't want to end up with nasty bugs like https://github.com/elastic/elasticsearch/issues/19009 where we don't pass enough information to the unmapped aggregation object and if they are chosen as the template for reducing they end up producing incorrect output. Could we at least add a test to check that a search with a custom format over two indexes (one with the field mapped and one without) always returns the keys correctly formatted?